specification/dimensions/pricingunit.md

```diff
@@ -0,0 +1,31 @@
+# Pricing Unit
+
+The Pricing Unit represents a provider specified pricing measurement unit, which forms the foundation for determining unit prices of utilizing a particular resource and/or service. Common examples include the number of hours a compute appliance was running for a given period, the number of gigabytes * the number of hours (gigabyte-hours) for a storage appliance, or a cumulative count of requests for a network appliance or API service. For certain services providers can choose block pricing i.e., million requests as a base unit.
```

Comment:
```suggestion
The Pricing Unit is a measurement unit specified by the provider, serving as the basis for calculating unit prices associated with using a specific resource and/or service. Examples include the duration a compute appliance is active, the product of gigabytes and hours (gigabyte-hours) for a storage appliance, or the total count of requests for a network appliance or API service. In some cases, providers may opt for block pricing, such as pricing per million requests as the pricing unit. The Pricing Unit, in conjunction with [Quantity In Pricing Unit](#quantityinpricingunit), defines and describes the measurement method for determining costs.
```

Reply:
My suggestion only.. use all/parts if it makes sense to do so.

Reply:
I like this.  I'm wondering how serious @flanakin is about the word "basis" vs "foundation", but I'm inclined to accept this as is and then we can discuss that part.  Thoughts?

Reply:
Sounds very robust, the wording is clear and the main message stays the same 👍

Reply:
I'd like to challenge the term `calculating` :). I feel like `determining` is a better fit in this context.

Reply:
@ijurica - you are referring to the fact that `Pricing Unit` is more of an information than a `p * q = c` dimension?

Reply:
Costs are usually calculated and unit prices determined/specified therefore, in this particular sentence, I would prefer the term **determining** over **calculating**. 

_The Pricing Unit is a measurement unit specified by the provider, serving as the basis for **calculating** unit prices associated with using a specific resource and/or service._

Reply:
Aligned during the call that we should determining 👍